Seafile.json

```
@@ -0,0 +1,34 @@
+{
+    "Seafile": {
+        "containers": {
+            "seafile": {
+                "image": "seafileltd/seafile",
+                "launch_order": 1,
+                "ports": {
+                    "80": {
+                        "description": "Seafile WebUI port. Suggested default: 8080",
+                        "host_default": 8080,
+                        "label": "WebUI port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/shared": {
+                        "description": "Choose a Share for Seafile. Eg: create a Share called seafile-all for this purpose alone.",
+                        "label": "Storage",
+                        "min_size": 1073741824
+                    }
+                }
+            }
+        },
+            "version": "latest",
+        "description": "Secure file sharing and hosting. <p>Based on the official Seafile docker image: <a href='https://hub.docker.com/seafileltd/seafile' target='_blank'>https://hub.docker.com/seafileltd/seafile</a>.",
```

I made a typo in the URL... sorry for the mistake.

```suggestion
        "description": "Secure file sharing and hosting. <p>Based on the official Seafile docker image: <a href='https://hub.docker.com/r/seafileltd/seafile' target='_blank'>https://hub.docker.com/r/seafileltd/seafile</a>.",
```